src/tigris.ts

```diff
@@ -168,8 +168,8 @@ export class Tigris {
 				config.serverUrl.includes("127.0.0.1") ||
 				config.serverUrl.includes("0.0.0.0:") ||
 				config.serverUrl.includes("[::1]")) &&
-			config.clientId === undefined &&
-			config.clientSecret === undefined
+			!config.clientId &&
```

Comment:
It should actually be undefined.

Reply:
Netlify for example overrides the values if not defined in the dev environment from the build environment variables. In that case, it will pass the client-id/client-secret from prod. I am not sure if this is a valid practice to pass empty clientId/clientSecret but seems like there is no other way.

Reply:
I didn't need the empty clientID/clientSecret after this https://github.com/tigrisdata/tigris-netlify-ecommerce/pull/6